targets/hasura/migrations/1600793735536_create_table_public_document_relations/up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE "public"."document_relations"(
```

Comment:
can you merge all the migrations into one file and rename it like xxxxxx_document_relations ?

Reply:
sure